content/doc/book/using/using-agents.adoc

```diff
@@ -0,0 +1,195 @@
++
+[source,bash]
+----
+docker run -d --rm --name=agent1 -p 22:22 \
+-e "JENKINS_AGENT_SSH_PUBKEY=[your-public-key]" \
+jenkins/ssh-agent:alpine
+----
++
+[NOTE]
+====
+* Remember to replace the tag [your-public-key] for your own SSH *public* key.
+* Your public key in this example is: `cat ~/.ssh/jenkins_agent_key.pub`
+====
+2. Now run the following command to update the container environment:
++
+[source,bash]
+----
+docker exec  agent1 sh -c "env | egrep -v "^(HOME=|USER=|MAIL=|LC_ALL=|LS_COLORS=|LANG=|HOSTNAME=|PWD=|TERM=|SHLVL=|LANGUAGE=|_=)" >> /etc/environment"
+----
++
+[NOTE]
+====
+The step 2 is necessary because the image is configured to reject changes to environment variables.
+when the link:https://github.com/jenkinsci/docker-ssh-agent/issues/33[issue #33] is fixed, we can ignore this step.
+====
+3. Now the container `agent1` is running. +
+Hint: the command `docker ps` can be used to check if the container is running as expected.
+
+=== Setup up the agent1 on jenkins.
+
+1. Go to your Jenkins dashboard;
+2. Go to `Manage Jenkins` option in main menu;
+3. Go to `Manage Nodes and clouds` item;
++
+image:node/node-1.png[Manage node menu]
+
+4. Go to `New Node` option in side menu;
+5. Fill the Node/agent name and select the type; (e.g. Name: agent1, Type: Permanent Agent)
+6. Now fill the fields:
+** Remote root directory; (e.g.: /home/jenkins )
+** label; (e.g.: agent1 )
+** usage; (e.g.: only build jobs with label expression...)
+** Launch method; (e.g.: Launch agents by SSH )
+*** Host; (e.g.: localhost or your IP address )
+*** Credentials; (e.g.: jenkins )
+*** Host Key verification Strategy; (e.g.: Manually trusted key verification ... )
+    image:node/node-2.png[node create form]
+7. Press the button save and the agent1 will be registered, but offline. Click on it.
++
+image:node/node-3.png[node offline]
+8. Now press the button `Launch agent` and wait some seconds, then you should receive +
+the message: `Agent successfully connected and online` on the last log line.
++
+image:node/node-4.png[Agent successfully connected]
+
+=== Delegating the first job to agent1
+
+1. Go to your jenkins dashboard;
+2. Select `New Item` on side menu;
+3. Enter a name. (e.g.: First Job to Agent1)
+4. Select the `Freestyle project` and press OK;
+5. Check the option: `Restrict where this project can be run`;
+6. Fill the field: label with the agent1 label; (e.g.: agent1)
++
+image:node/node-5.png[Agent job 1]
+
++
+[NOTE]
+====
+Be careful with white spaces before or after the label.
+====
+
+7. Now Select the option `Execute shell` at Build Section;
++
+image:node/node-6.png[Agent job 2]
+
+8. Add the command: `echo $NODE_NAME` in the `Command` field of the `Execute shell` step  and the name +
+of the agent will be printed inside the log when this job is run;
+9. press the save button and then select the option `Build Now`;
+10. Wait some seconds ant the go to `Console Output` page
++
+image:node/node-7.png[Agent job 3]
```

Comment:
Limit width of image, allow user to open full size image in another window or tab
```suggestion
image:node/node-7.png[Agent job 3,650]
```